osquery/tables/system/windows/system_util.h

```diff
@@ -24,19 +24,21 @@
 
 namespace osquery {
 namespace tables {
- 
+
 class WmiResultItem {
  public:
-  explicit WmiResultItem(IWbemClassObject *result) : result_(result) {};
+  explicit WmiResultItem(IWbemClassObject* result) : result_(result){};
```

Comment:
I think enough of this needs  refactor, that it would make sense to roll all of this into a WMI class, and I'll add the comments describing each at that point.